gammapy/maps/hpxnd.py

```diff
@@ -648,3 +649,28 @@ def get_angle(x, t):
         ax.coords.grid(color='w', linestyle=':', linewidth=0.5)
 
         return fig, ax, p
+
+    def make_region_mask(self, region, inside=True):
+        """Create a mask of a given region
+
+        TODO: implement list of region for each axis
+
+        Parameters
+        ----------
+        region :  `~regions.SkyRegion` object
+            A region on the sky could be defined in pixel or sky coordinates.
+            Note that `~regions.PixelRegion` does not work for healpix maps.
+        inside : bool
+            Output map is True inside the input region if inside is set to True and False outside and conversely.
+
+        Return
+        ------
+        mask_map : `~gammapy.maps.HpxNDMap`
+            the mask map
+        """
+        mask = self.geom.get_region_idx(region)
+        if inside is False:
+            np.logical_not(mask,out=mask)
+
+        # TODO : update meta table to include something about the region used for mask creation?
```

Comment:
Suggest to remove this TODO.

Adding such book-keeping could be done all throughout Gammapy, but would be more than double of actually writing the actual code and needs a lot of thought. Let's write code first, and then think about such provenance later.

Reply:
OK will remove the TODO. 
I put it to remember that at some point we need to do something here with the metadata.